apiserver/facades/client/applicationoffers/applicationoffers.go

```diff
@@ -513,3 +513,35 @@ func (api *OffersAPI) oneRemoteApplicationInfo(urlStr string) (*params.RemoteApp
 		IconURLPath:      fmt.Sprintf("rest/1.0/remote-application/%s/icon", url.ApplicationName),
 	}, nil
 }
+
+// DeleteOffers removes the offers specified by the given URLs.
+func (api *OffersAPI) DeleteOffers(args params.DestroyApplicationOffers) (params.ErrorResults, error) {
```

Comment:
Should this be called RemoveOffers or just Remove (since it's on the ApplicationOffers facade anyway)? We're already kind of stuck with Remove and Destroy, but if we can avoid adding Delete as well we should. (I'm not sure whether we already have it in other places in the API.)

Reply:
On machine manager, we still call it "DestroyMachine", so I think DeleteOffers is ok.
I'll rename to DestroyOffers to be consistent.